provisions/group_vars/all

```diff
@@ -9,3 +9,5 @@ cccp_source_branch: master
 public_registry: registry.centos.org
 
 enable_epel: true
+
+test_nfs_share: 172.29.32.44:/nfsshare
```

Comment:
@navidshaikh CI nodes cannot access devcloud. So, we need to setup another CI node, if it's needed.